protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1973,6 +1981,22 @@ message MarketConnection {
   PageInfo page_info = 2;
 }
 
+// Request for listing successor markets
+message ListSuccessorMarketsRequest {
+  // Market ID that is a member of the succession chain. This can be the original market
+  // or any subsequent child market that succeeded.
+  string market_id = 1 [(google.api.field_behavior) = REQUIRED];
+  // Flag to indicate whether or not to include the full succession chain, or only list
+  // the children of the given market ID. If true, the full succession chain
+  bool full_history = 2;
+}
+
+// Response from listing successor markets request.
+message ListSuccessorMarketsResponse {
+  // List of markets and corresponding page information.
```

Comment:
"list of successor markets" specifically?

Reply:
There is also no page information returned. I guess we've decided not to because it there will never been that many?